packaging/docs-weapons.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import json
+from collections import OrderedDict
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--input', type=str, required=True)
+    parser.add_argument('--output', type=str, required=True)
+    args = parser.parse_args()
+
+    inputFile = open(args.input, "r")
+    jj = json.loads(inputFile.read())
+    inputFile.close()
+
+    weaponsByNamespace = OrderedDict()
+    for currentWeapon in jj["WeaponTypes"]:
+        if currentWeapon["Namespace"] in weaponsByNamespace: weaponsByNamespace[currentWeapon["Namespace"]].append(currentWeapon)
+        else: weaponsByNamespace[currentWeapon["Namespace"]] = [ currentWeapon ]
+
+    version = jj["Version"]
+
+    outputFile = open(args.output, "w")
+    outputFile.write("#### This documentation is aimed at modders. It displays a template for weapon definitions " +
+				"as well as its contained types (warheads and projectiles) with default values and developer commentary. " +
+				"Please do not edit it directly, but add new `[Desc(\"String\")]` tags to the source code.\n\n")
+
+    outputFile.write(f"This file has been automatically generated for version **{version}** of OpenRA.")
+    outputFile.write('\n\n#\n\n')
+
+    for namespace in weaponsByNamespace:
+        outputFile.write(f'## {namespace}\n')
```

Comment:
`outputFile.writeline` and drop the explicit `\n`'s.

Reply:
My google search found no evidence that there is a `.writeline` method :?
I found `.writelines`, which I guess I could use 🤷‍♂️